Practice-Problems/odd-up-even-down/main.js

```diff
@@ -0,0 +1,17 @@
+function oddUpEvenDown(array) {
+  var result = [];
+  for (var i = 0; i<array.length; i+=2){
+    array[i] = array[i] + 1;
+    array[i+1] = array[i+1] - 1;
```

Comment:
Why are you trying to do two items at once in your code here? If you do these values one at a time, there shouldn't be a need for conditionals to check if `array[i + 1]` is a value or not.